base/methodshow.jl

```diff
@@ -118,6 +118,16 @@ end
 default_methodloc(method::Method) = method.file, method.line
 const methodloc_callback = Ref{Function}(default_methodloc)
 
+# This function does the method location updating
+function updated_methodloc(m)
+    file, line = invokelatest(methodloc_callback[], m)
+    if file !== nothing && isdefined(@__MODULE__, :Sys)
```

Comment:
```suggestion
    if file !== nothing && Sys.BUILD_STDLIB_PATH != Sys.STDLIB
```
Is Sys ever not defined?

And should we put this inside `default_methodloc`, or do we want to do this unconditionally?

Reply:
It's not defined when this function gets defined and I was worried if there was some error before `Sys` gets defined then it would crash when trying to print some method.

Reply:
I think we want to do this unconditionally.